migrations/20230429053842_add_donation_relation_to_donation_wishes/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AddForeignKey
+ALTER TABLE "donation_wishes" ADD CONSTRAINT "donation_wishes_donation_id_fkey" FOREIGN KEY ("donation_id") REFERENCES "donations"("id") ON DELETE SET NULL ON UPDATE CASCADE;
```

Comment:
For to ensure the new foreign key will be applied, please add the following on top of the file. 
```
-- Ensure that there are no orphaned donation wishes
DELETE FROM "donation_wishes" WHERE "donation_id" NOT IN (SELECT "id" FROM "donations");
```